modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -49,6 +49,53 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
         }
       }
     }
+
+    // Text Background Color.
+    if (!empty($this->configuration['bg_text_color'])) {
+      $bg_text_color = $row->getSourceProperty($this->configuration['bg_text_color']);
+      $bg_text_color_mapping = [
+        'bg-transparent' => 'transparent',
+        'bg-trans-white' => 'light',
+        'bg-trans-sky' => 'light',
+        'bg-trans-arizona-blue' => 'dark',
+        'bg-trans-black' => 'dark',
```

Comment:
This paragraph type doesn't have sky and blue options in QS1.

```suggestion
        'bg-transparent' => 'transparent',
        'bg-trans-white' => 'light',
        'bg-trans-black' => 'dark',
```